pmultiqc/modules/quantms/quantms.py

```
@@ -1838,6 +2015,261 @@ def getSpectrumCountAcrossRep(condition_count_dict: dict):
+        if "PeptideEvidenceRef_location" in mzid_table.columns:
+            mzid_table["PeptideEvidenceRef_location"] = mzid_table["PeptideEvidenceRef_location"].apply(parse_location)
+        
+        mzid_table["stand_spectra_ref"] = mzid_table.apply(lambda x: self.file_prefix(x.location), axis=1)
+        mzid_table["Modifications"] = mzid_table["Modification"].apply(lambda x: Process_Modification(x))
+
+        if "cross-link spectrum identification item" in mzid_table.columns:
+            mzid_table["CrossLinked_Peptide"] = ~mzid_table["cross-link spectrum identification item"].isna()
+        else:
+            mzid_table["CrossLinked_Peptide"] = False
+
+        mzid_table["pep_to_prot_unique"] = mzid_table.groupby(["spectrumID", "PeptideSequence"])["accession"].transform(lambda x: True if len(set(x)) == 1 else False)
```

Ruff

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)